### PR TITLE
Create Ask New Command documentation

### PR DESCRIPTION
**Pull request by ChihYing7509**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: docs/concepts/New-Command.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+**OPTIONS DESCRIPTION:**
+
+**template-url**: Optional. Specify the URL of a Git repository that contains a skill template.
+
+**profile**: Optional. Specify a profile name to be used. Defaults to use `default` as the profile name, if this option or environmental variable `ASK_DEFAULT_PROFILE` is not set.
+
+**debug**: Optional. Appends a debug message to the standard error.
+
+
+## WORKFLOW:
```
Comment (kakhaUrigashvili) [inline docs/concepts/New-Command.md]: do we need those details?

File: docs/concepts/New-Command.md [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+│   └── interactionModels 
+│       └── custom
+│           └── en-US.json
+│   └── skill.json 
+├── ask-resources.json
+├── .gitignore
+```
+The following list explains each part of the skill project:
+  
+  **lambda** – A folder that contains the source code for the skill's AWS Lambda function. The files contained here depend on the runtime for the skill.
```
Comment (kakhaUrigashvili) [inline docs/concepts/New-Command.md]: there is no lambda in the example above. I thought either should be lambda or code folder but not both?

Comment (ChihYing7509) [reply docs/concepts/New-Command.md]: Yes, it's either code or lambda, now all templates are updated to "lambda", so only lambda will be kept here.

File: docs/concepts/New-Command.md [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+* **AWS Lambda**
+  * Dependency: AWS-SDK
+  * Description: Deploys updated Lambda function code by replicating the existing CLI V1 functionality. Function ARN will be returned back to the CLI and further updated in the skill manifest.
+* **AWS CloudFormation**
+  * Dependency: AWS-SDK
+  * Description: Provides CloudFormation stack template(s) to deploy skill backends, making it easy to standardize skill infrastructures. Multiple regions can be configured. 
+  Function ARNs need to be the output in each yaml template file.
+* **Custom Deployer**
+  * Dependency: Custom Shell Script
+  * Description: Allows all other serverless deployment flow using CLI deploy logic.
```
[the remark this answers is not in the document]
Comment (ChihYing7509) [reply docs/concepts/New-Command.md]: I removed this deployer for now

**Pull request merged**